src/common/components/ui/PreviewBadge.tsx

```diff
@@ -27,7 +27,12 @@ const PreviewBadge: React.FC<
   }
 
   return (
-    <Label className={clsName} color={color}>
+    <Label
+      className={clsName}
+      color={color}
+      onClick={(e) => e.preventDefault()}
+      style={{ cursor: 'pointer' }}
```

Comment:
Why cursor is pointer but clicking the badge does nothing?

Reply:
1. The badge should be with the "I" icon in it
2. The badges should be clickable and show the popover
3. The cursor should change to a pointer when hovering the badge